### ISSUE TITLE
Настраиваемый типограф для текстового поля

### ISSUE THREAD

**Issue by krylos** [instantsoft/icms2#712]
При добавлении текстового поля можно включить типограф.
Добавить настройки, чтобы типограф можно было отключать для определенных групп пользователей, или наоборот указать для каких групп будет будет типограф.

Что получится: допустим добавляем текстовое поле и типограф включается, но для администраторов отключен. Так текстовое поле добавленное админом будет публиковаться без обработки типографом.

Сейчас же если хочется дать админу возможность опубликовать что то без типографа приходится создавать отдельное поле доступное только админу и выводить там то что нужно безтипографа.

**Comment by nomad48** [comment]
да, было бы не плохо, а то iframe вставки от видео режет, но можно было бы админу разрешить вставлять.

**Comment by vlad7090** [comment]
согласен полностью

**Comment by fuzegit** [comment]
#923
